packages/amplify-category-auth/src/provider-utils/awscloudformation/index.ts

```diff
@@ -412,48 +427,40 @@ const getCognitoOutput = amplifyMeta => {
   return cognitoOutput;
 };
 
-const openAdminUI = async (context, appId, region) => {
+const openAdminUI = async (context: $TSContext, appId: any, region: any): Promise<any> => {
```

Comment:
I think appId and region are both strings. We should have some type for both of these to have some validation for each of these params.